modules/login/php/module.class.inc

```diff
@@ -47,4 +49,22 @@ class Module extends \Module
     {
         return "Login";
     }
+
+    /**
+     * Check whether a user is logined or not, if yes, redirect to the base URL
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface the outgoing PSR7 response
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        $user = \User::singleton();
```

Comment:
The base router includes a "user" attribute on the request which is a (slightly) better place to get the user than a singleton.